selfie_segmentation/index.html

```diff
@@ -0,0 +1,178 @@
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <title>WebNN Selfie Segmentation Example</title>
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css"
+    integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z" crossorigin="anonymous">
+  <link href='../common/css/style.css' rel='stylesheet'>
+  <link rel="icon" type="image/x-icon" href="../common/img/favicon/favicon.ico">
+  <link rel="apple-touch-icon" sizes="180x180" href="../common/img/favicon/apple-touch-icon.png">
+  <link rel="icon" type="image/png" sizes="32x32" href="../common/img/favicon/favicon-32x32.png">
+  <link rel="icon" type="image/png" sizes="16x16" href="../common/img/favicon/favicon-16x16.png">
+  <link rel="manifest" href="../common/img/favicon/site.webmanifest">
+</head>
+
+<body>
+  <nav class="navbar navbar-expand-md navbar-dark">
+    <a href="../index.html">
+      <div class="logosvg" id="logosvg"></div>
+    </a>
+    <span class="navbar-brand">Selfie Segmentation</span>
+    <!-- <ul class="navbar-nav"></ul> -->
+  </nav>
+  <div class='container' id='container'>
+    <div class="card mt-2 mb-2">
+      <div class="card-body">
+        <div class="row mb-2 align-items-center justify-content-md-left">
+          <div class="col col-md-1">
+            <span>Delegate</span>
+          </div>
+          <div class="col-md-auto">
+            <div class="btn-group-toggle" data-toggle="buttons" id="delegateBtns">
+              <label class="btn btn-outline-info custom active">
+                <input type="radio" name="delegate" id="xnnpack" autocomplete="off" checked>XNNPACK (SIMD)
+              </label>
+              <label class="btn btn-outline-info custom">
+                <input type="radio" name="delegate" id="webnn" autocomplete="off">WebNN (CPU)
```

Comment:
Should we just leave a checkbox that enable webnn delegate? Like the tf.js e2e benchmark does. Because even when WebNN delegate is enabled, the XNNPACK WASM backend is still used.

WDYT?

Reply:
Yes, updated.